FunctionSignatureMap.php

```diff
@@ -9933,7 +9934,7 @@
 'RecursiveDirectoryIterator::current' => ['mixed'],
 'RecursiveDirectoryIterator::getATime' => [''],
 'RecursiveDirectoryIterator::getBasename' => [''],
-'RecursiveDirectoryIterator::getChildren' => ['object'],
+'RecursiveDirectoryIterator::getChildren' => ['RecursiveDirectoryIterator'],
```

Comment:
https://www.php.net/manual/en/recursivedirectoryiterator.getchildren.php#refsect1-recursivedirectoryiterator.getchildren-returnvalues

> The filename, file information, or $this depending on the set flags. See the FilesystemIterator constants.

I'm assuming php.net is wrong and the text description was copied from another method